src/Console/Command/DescribeCommand.php

```diff
@@ -225,7 +225,13 @@ private function describeRule(OutputInterface $output, $name)
                 $old = $codeSample->getCode();
                 $tokens = Tokens::fromCode($old);
                 if ($fixer instanceof ConfigurableFixerInterface) {
-                    $fixer->configure($codeSample->getConfiguration());
+                    $configuration = $codeSample->getConfiguration();
+
+                    if (null === $configuration) {
```

Comment:
no, it's the interface of `CodeSample` for non-configurable fixers and **not** passing any configuration to the fixer (when you pass `{"fixer": true}` in config)